demos/toolbar/default-toolbar.html

```diff
@@ -0,0 +1,68 @@
+<!DOCTYPE html>
+<!--
+  Copyright 2016 Google Inc. All rights reserved.
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+      https://www.apache.org/licenses/LICENSE-2.0
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License
+-->
+<html>
```

Comment:
could we add `class="mdc-typography"` here? This way the fonts are styled with Roboto and the demo has a more cohesive look and feel. Most of our demos that contain a more-than-trivial amount of textual content have this class applied, e.g. https://github.com/material-components/material-components-web/blob/master/demos/select.html